Source/Urho3D/CSharp/Managed/Scene/Node.cs

```diff
@@ -43,22 +45,32 @@ public partial class Node
         /// Get first occurrence of a component type
         /// </summary>
         /// <typeparam name="T"></typeparam>
-        /// <returns></returns>
+        /// <returns>Found component or null.</returns>
         public T GetComponent<T>() where T: Component
         {
             return (T)GetComponent(typeof(T).Name);
         }
 
         /// <summary>
-        /// get all components of a type
+        /// Get all components of a type
         /// </summary>
         /// <typeparam name="T"></typeparam>
-        /// <returns></returns>
+        /// <returns>List of found component or null.</returns>
```

Comment:
This comment is not true. It always returns a non-null list that would be empty if no components are found.